samples/snippets/pom.xml

```diff
@@ -41,8 +39,9 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-securitycenter-settings</artifactId>
+      <!-- TODO: REMOVE VERSION SPECIFIED BELOW WHEN THE CLIENT GOES GA-->
+      <version>0.2.1-SNAPSHOT</version>
```

Comment:
this should be the latest release version and not snapshot

Reply:
fixed